### DISCUSSION TITLE
Unreachable code warning

### DISCUSSION THREAD

**Discussion by karthiksathishjeemain** [Cyfrin/foundry-full-course-cu#1062] [Q&A]
![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/115690993/aef743e1-53fe-400d-8ac2-cdb2c9649a89)
How to solve this warning?

**Comment by Niferu** [comment]
Hi @karthiksathishjeemain 

Please link your repo

**Comment by DukeLurkin** [comment]
Ever get an answer to this?

**Comment by smahdi-dev** [comment]
@karthiksathishjeemain did you solve this??
